src/server.h

```diff
@@ -870,6 +870,7 @@ typedef struct client {
     size_t sentlen;         /* Amount of bytes already sent in the current
                                buffer or object being sent. */
     time_t ctime;           /* Client creation time. */
+    time_t duration;        /* Current command duration. Used for measuring latency of blocking/non-blocking cmds */
```

Comment:
@filipecosta90 Consider using a `long` here as used elsewhere for ustime, `time_t` is confusing as it represents seconds-since-epoch.

Reply:
fixed in the last commit @yossigo :)